torch/onnx/_exporter_states.py

```diff
@@ -25,6 +26,12 @@ class SymbolicContext:
         onnx_block (_C.Block): Current ONNX block that converted nodes are being appended to.
     """
 
+    @_deprecation.deprecated(
+        "1.13",
+        "TBD",
+        # TODO(justinchuby): Fix the instruction when GraphContext is public.
+        "remove the 'ctx' argument and annotate 'g: GraphContext' instead",
+    )
```

Comment:
Will `SymbolicContext` be deprecated?

Reply:
I intended to deprecate SymbolicContext. @BowenBao @shubhambhokare1 @thiagocrepaldi any thoughts?

Reply:
Sounds good, since all previous functionalities can be achieved through new GraphContext.